src/C_wrapper.jl

```diff
@@ -83,23 +99,32 @@ function _Eval_Jac_G_CB(
     x = unsafe_wrap(Array, x_ptr, Int(n))
     rows = unsafe_wrap(Array, iRow, Int(nele_jac))
     cols = unsafe_wrap(Array, jCol, Int(nele_jac))
+    new_x = convert(Bool, x_new)
     if values_ptr == C_NULL
-        prob.eval_jac_g(x, rows, cols, nothing)
+        if prob.expose_xnew
+            prob.eval_jac_g(x, new_x, rows, cols, nothing)
+        else
+            prob.eval_jac_g(x, rows, cols, nothing)
+        end
     else
         values = unsafe_wrap(Array, values_ptr, Int(nele_jac))
-        prob.eval_jac_g(x, rows, cols, values)
+        if prob.expose_xnew
+            prob.eval_jac_g(x, new_x, rows, cols, values)
+        else
+            prob.eval_jac_g(x, rows, cols, values)
+        end
     end
     return Cint(1)
 end
 
 function _Eval_H_CB(
     n::Cint,
     x_ptr::Ptr{Float64},
-    ::Cint,
+    x_new::Cint,
     obj_factor::Float64,
     m::Cint,
     lambda_ptr::Ptr{Float64},
-    ::Cint,
+    new_lambda::Cint,
```

Comment:
```suggestion
   ::Cint,
```
We don't name arguments if they aren't used. But if we're going th `x_new` route, should we not include `new_lambda` as well?

Reply:
Yes to be consistent, we should also expose new_lambda but I can't imagine many users using the exact hessian routines